components/plurals/src/provider/rules/mod.rs

```diff
@@ -146,21 +146,25 @@
 //! [`PluralCategory::Other`]: super::PluralCategory::Other
 //! [`PluralOperands`]: super::PluralOperands
 //! [`PluralRules::categories`]: super::PluralRules::categories
-//! [`PluralRuleType::Cardinal`]: super::PluralRuleType::Cardinal
+//! [`PluralRuleType::Cardinal`]: crate::PluralRuleType::Cardinal
 //! [`Rule`]: super::rules::reference::ast::Rule
 //! [`Rules`]: super::rules::reference::ast::Rule
 //! [`Condition`]: super::rules::reference::ast::Condition
 //! [`Sample`]: super::rules::reference::ast::Samples
 //! [`AST`]: super::rules::reference::ast
 
-#[doc(hidden)]
+pub mod runtime;
+
+// The reference module is used internally, but is only needed externally in datagen mode
```

Comment:
this applies to a lot of code here. as all of this is unstable, just make it public?

Reply:
Yeah, I started doing this as a way to figure out what can be mvoed to datagen, but it's messy. I can get rid of the conditionality